# This is a BitKeeper generated diff -Nru style patch.
#
# ChangeSet
#   2004/06/01 02:58:12+02:00 kaber@trash.net 
#   [NETFILTER]: Change permissions of /proc/net/ip_conntrack to 0440
# 
# net/ipv4/netfilter/ip_conntrack_standalone.c
#   2004/06/01 02:58:02+02:00 kaber@trash.net +1 -1
#   [NETFILTER]: Change permissions of /proc/net/ip_conntrack to 0440
# 
diff -Nru a/net/ipv4/netfilter/ip_conntrack_standalone.c b/net/ipv4/netfilter/ip_conntrack_standalone.c
--- a/net/ipv4/netfilter/ip_conntrack_standalone.c	2004-06-02 03:23:39 +02:00
+++ b/net/ipv4/netfilter/ip_conntrack_standalone.c	2004-06-02 03:23:39 +02:00
@@ -502,7 +502,7 @@
 	if (ret < 0)
 		goto cleanup_nothing;
 
-	proc = proc_net_create("ip_conntrack",0,list_conntracks);
+	proc = proc_net_create("ip_conntrack", 0440, list_conntracks);
 	if (!proc) goto cleanup_init;
 	proc->owner = THIS_MODULE;
 
